lib/compat/wordpress-6.0/block-template-utils.php

```diff
@@ -62,6 +62,16 @@ function gutenberg_generate_block_templates_export_file() {
 		wp_json_encode( $tree->get_data(), JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE )
 	);
 
+	$zip->addFromString(
+		'theme/index.php',
+		'<?php // This file is needed to make the theme work, but it doesn\'t do anything.'
+	);
+
+	$zip->addFromString(
+		'theme/style.css',
+		file_get_contents( get_stylesheet_directory() . '/style.css' )
+	);
```

Comment:
I think we can use [`ZipArchive::addFile`](https://www.php.net/manual/en/ziparchive.addfile.php) here. It is what core uses in privacy tools export.